queries.sql

```diff
@@ -2,9 +2,10 @@
 
 SELECT * from animals WHERE name LIKE '%mon';
 
-SELECT name FROM animals WHERE YEAR(date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
 
-SELECT name FROM animals WHERE neutered = 1 AND escape_attempts < 3;
```

Comment:
- [ ] This query should be written as the code below. currently what you have is `neutered = 1`
```sql
SELECT * FROM animals WHERE neutered = 'true';
```